integreat_cms/cms/templates/content_language_switcher.html

```diff
@@ -0,0 +1,47 @@
+<!--generic template for language switcher-->
+<!--please set variable "target" when including this code-->
+{% load content_filters %}
+{% load i18n %}
+
+<div id="language-switcher" class="flex flex-col z-10 text-lg">
+    <a id="language-switcher-current" href="{% url target region_slug=region.slug language_slug=language.slug page_id=content_id %}" class="font-bold flex justify-between bg-white py-2 px-4 rounded shadow hover:bg-gray-200 border-2 border-white hover:border-gray-200">
+        Weitere Sprachen
+    </a>
+    <div id="language-switcher-list" class="h-0 hidden">
+        {% for other_language in languages|sort_languages:language|slice:"4:" %}
+            <a href="{% url target region_slug=region.slug language_slug=other_language.slug page_id=content_id %}" class="block flex justify-between whitespace-nowrap {% if forloop.last %} rounded-b {% endif %} shadow px-4 py-3 text-gray-800 hover:bg-gray-200 bg-white">
+                {% if other_language in page.languages %}
+                    {% get_translation page_form.instance other_language.slug as other_page_translation %}
```

Comment:
```suggestion
                    {% get_translation instance other_language.slug as other_page_translation %}
```
If you want to, you could also generalize this template a bit more to make it possible to use it in the event form and poi form as well. In this case, you would have to pass the form instance as `instance` when including this template (similar to `target`).